licence.json

```diff
@@ -0,0 +1,79 @@
+{
+  "id": 99999,
+  "type": "STD | HDC",
+  "subType": "AP | PSS | AP+PSS",
+  "policyVersion": "2.1",
+  "version": "1.4",
+  "statusCode": "ACTIVE",
+  "nomsId": "A9999AA",
+  "bookingId": 989898,
+  "crn": "X12444",
+  "approvedByUsername": "X33221",
+  "approvedDate": "2023-09-27T16:59:04.886Z",
+  "createdByUsername": "X12333",
+  "createdDate": "2023-09-27T16:59:04.886Z",
+  "updatedByUsername": "X34433",
+  "updatedDate": "2023-09-27T16:59:04.886Z",
+  "isInPssPeriod": true,
+  "appointment": {
+    "contact": "Duty officer",
+    "time": "2023-09-27T16:59:04.886Z",
+    "address": "Manchester Probation Service, Unit 4, Smith Street, Stockport, SP1 3DN",
```

Comment:
just wondering if it's worth having an address type? I know this is how we store addresses in cvl but at the moment this doesn't handle a "," in the address

Reply:
+1

Reply:
Yes, agree but that's a slightly wider problem that maybe it would be better to resolve outside the scope of this work.

I think lets remove it for now and we can add it back in once we've resolved that issue